sp/src/game/server/ez2/npc_husk_base.h

```diff
@@ -1107,6 +1112,24 @@ void CAI_BaseHusk<BASE_NPC>::RefreshCognitionFlags()
 	}
 }
 
+//-----------------------------------------------------------------------------
+//-----------------------------------------------------------------------------
+template <class BASE_NPC>
+void CAI_BaseHusk<BASE_NPC>::CheckPlayerAllied()
+{
+	this->m_bAlliedWithPlayer = false;
+
+	if ( this->HasPassiveTargets() )
+	{
+		// Check if one of our passive targets is a player
+		for (int i = 0; i < this->m_PassiveTargets.Count(); i++)
+		{
+			if (m_PassiveTargets[i] && m_PassiveTargets[i]->IsPlayer())
+				this->m_bAlliedWithPlayer = true;
```

Comment:
Why does a passive target being the player necessarily mean they are allied? Couldn't they be neutral?

Reply:
A passive target is normally someone who has befriended the husk. Husks are always friendly to passive targets unless the passive target starts attacking them.